stl/stari/gyf/src/hello.cpp

```diff
@@ -0,0 +1,7 @@
+#include <iostream>
+
+int main(void)
+{
```

Comment:
Please write code according to Google Code Style

1: `{` do not need to start a new line
2: Indent two spaces

```cpp
int main(void)
{
    std::cout << "Hello World" << std::endl;
    return 0;
}
```


=> 

```cpp
int main(void) {
  std::cout << "Hello World" << std::endl;
  return 0;
}
```

Reply:
> Please write code according to Google Code Style
> 
> 1: `{` do not need to start a new line 2: Indent two spaces
> 
> ```c++
> int main(void)
> {
>     std::cout << "Hello World" << std::endl;
>     return 0;
> }
> ```
> 
> =>
> 
> ```c++
> int main(void) {
>   std::cout << "Hello World" << std::endl;
>   return 0;
> }
> ```

The style has been fixed, thank u!